api/db/changelog/db.changelog-172-backfill-user-access-module.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="yonghao" id="changelog-172-backfill-user-access-module">
+    <addUniqueConstraint tableName="user_access_module"
+      columnNames="user_id, access_module_id"
+      constraintName="user_access_module_unique_name_per_user_per_module"/>
+    <!--Remove everything before backfill-->
+    <sql>
+      DELETE from user_access_module WHERE true
```

Comment:
Please move this before the unique constraint - I can't run it because I already have a duplication.

Reply:
right! Done